### ISSUE TITLE
idea - give kudos marketplace same treatment as kudos profile

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#2847]
random idea..

what if the kudos in the marketplace ..
<img width="811" alt="screen shot 2018-11-15 at 11 27 32 am" src="https://user-images.githubusercontent.com/513929/48574632-e6ec0500-e8cc-11e8-9dc4-b27eaa2267c2.png">

had the same treaetment that the kudos cards in the profile have

<img width="812" alt="screen shot 2018-11-15 at 11 26 11 am" src="https://user-images.githubusercontent.com/513929/48574635-e9e6f580-e8cc-11e8-96aa-00cac70bc85a.png">

basically you can eye ball how popular they are by seeing how many ppl have them (up to a point, in which we say PERSON X, Y, Z and 10 others)
